doc/modules/cassandra/pages/new/java11.adoc

```diff
@@ -26,11 +26,15 @@ Table 1 : Support Matrix for Java
 |===
 | |Java 8 (Run) |Java 11 (Run)
 |Java 8 (Build) |Supported |Supported
-|Java 11(Build) |Not Supported |Supported
+|Java 11(Build) |Not Supported |Experimental
 |===
 
-Essentially Apache 4.0 source code built with Java 11 cannot be run with
-Java 8. Next, we shall discuss using each of Java 8 and 11 to build and
+Apache 4.0 source code built with Java 11 cannot be run with
+Java 8.
+
+All binary releases are built with Java 8.
```

Comment:
?

Reply:
This is from the time they were 8.... check my previous comment for this doc. Maybe we just revert and skip updating this file, not sure